Pear/Controller/SchedulingTimeViewController.swift

```diff
@@ -489,19 +492,29 @@ class SchedulingTimeViewController: UIViewController {
         }
     }
 
-    // MARK: - Error Message
-    private func setupErrorMessageAlert() {
+    private func showErrorMessageAlertView() {
         errorMessageAlertView = MessageAlertView(
-            delegate: self,
             mainMessage: Constants.Alerts.NoTimesWork.message,
             actionMessage: Constants.Alerts.NoTimesWork.action,
-            dismissMessage: Constants.Alerts.NoTimesWork.dismiss
+            dismissMessage: Constants.Alerts.NoTimesWork.dismiss,
+            alertImageName: "sadPear",
+            removeFunction: { isDismiss in
```

Comment:
Is this the naming convention you use throughout the app? My instinct was that `dismiss` would be more idiomatic, but maybe it's clear in the broader context.